merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/remotes/postgres/CreateActivityTypeAction.java

```diff
@@ -1,18 +1,25 @@
 package gov.nasa.jpl.aerie.merlin.server.remotes.postgres;
 
 import gov.nasa.jpl.aerie.merlin.protocol.types.Parameter;
+import gov.nasa.jpl.aerie.merlin.protocol.types.ValueSchema;
 import org.intellij.lang.annotations.Language;
 
 import java.sql.Connection;
 import java.sql.PreparedStatement;
 import java.sql.SQLException;
 import java.util.List;
+import java.util.Optional;
+
+import static gov.nasa.jpl.aerie.merlin.server.http.ValueSchemaJsonParser.valueSchemaP;
 
 /*package-local*/ final class CreateActivityTypeAction implements AutoCloseable {
   private static final @Language("SQL") String sql = """
-    insert into activity_type (model_id, name, parameters, required_parameters)
-    values (?, ?, ?::json, ?::json)
-    on conflict (model_id, name) do update set parameters = ?::json, required_parameters = ?::json
+    insert into activity_type (model_id, name, parameters, required_parameters, return_value_schema)
+    values (?, ?, ?::json, ?::json, ?::json)
+    on conflict (model_id, name) do update
+      set parameters = ?::json,
+      required_parameters = ?::json,
+      return_value_schema = ?::json
```

Comment:
I wanted to suggest you rework this so you don't have to specify the values a second time for these, but apparently it's not possible :(

https://stackoverflow.com/questions/13586012/using-same-prepared-statement-parameter-multiple-times-in-a-sql